pkg/kubelet/qos/policy.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package qos
+
+import (
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+)
+
+type Tier int
+
+const (
+	PodInfraOomAdj int  = -999
+	BestEffort     Tier = iota
+	Burstable
+	Guaranteed
+)
+
+// IsBestEffort returns true if the pod is Best-Effort.
+func IsBestEffort(podSpec *api.PodSpec) bool {
```

Comment:
QoS classes are per-resource so we should specifically say that these functions are for memory. Maybe MemoryIsBestEffort(). In theory we can do it generically for any resource.